src/robusta/runner/config_loader.py

```diff
@@ -116,32 +120,81 @@ def __load_playbooks_repos(
         for playbook_package, playbooks_repo in playbooks_repos.items():
             try:
                 if playbooks_repo.pip_install:  # skip playbooks that are already in site-packages
-                    if playbooks_repo.url.startswith(GIT_SSH_PREFIX) or playbooks_repo.url.startswith(GIT_HTTPS_PREFIX):
-                        repo = GitRepo(playbooks_repo.url, playbooks_repo.key.get_secret_value(), playbooks_repo.branch)
-                        local_path = repo.repo_local_path
-                    elif playbooks_repo.url.startswith(LOCAL_PATH_URL_PREFIX):
-                        local_path = playbooks_repo.url.replace(LOCAL_PATH_URL_PREFIX, "")
-                    else:
-                        raise Exception(
-                            f"Illegal playbook repo url {playbooks_repo.url}. "
-                            f"Must start with '{GIT_SSH_PREFIX}', '{GIT_HTTPS_PREFIX}' or '{LOCAL_PATH_URL_PREFIX}'"
-                        )
-
-                    if not os.path.exists(local_path):  # in case the repo url was defined before it was actually loaded
-                        logging.error(f"Playbooks local path {local_path} does not exist. Skipping")
-                        continue
-
-                    # Adding to pip the playbooks repo from local_path
-                    subprocess.check_call([sys.executable, "-m", "pip", "install", "--no-build-isolation", local_path])
-                    playbook_package = self.__get_package_name(local_path=local_path)
+                    remove_pkg_path = None
+                    try:
```

Comment:
where is the except for this? (it could be something i dont see in git?
alsi there is a try a few lines up, why do we need another try?

Reply:
There is no `except`. There is a `finally` in line 174 below.